.github/workflows/auto_request_review.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Request Review
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review, reopened]
+
+jobs:
+  auto-request-review:
+    name: Auto Request Review
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Request review based on which files changed
+        uses: necojackarc/auto-request-review@v0.7.0
```

Comment:
to mitigate risk of account takeover, we could pin to the hash of the current version so that the action image is immutable.

Reply:
done. Testing now

Reply:
Testing complete